tests/unit/s2n_pq_cipher_suite_test.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+#include "tls/s2n_cipher_preferences.h"
+#include "tls/s2n_kex.h"
+
+#if !defined(S2N_NO_PQ)
+#define NUM_PQ_CIPHER_SUITES 3
+const struct s2n_cipher_suite *pq_cipher_suites[NUM_PQ_CIPHER_SUITES] = {
```

Comment:
Could you define an empty array when S2N_NO_PQ to avoid the #ifs in the test?

Reply:
Sure